### PR TITLE
chore: fix workspace selector's singout button styles affected by removing override in Nancy #447

### PR DESCRIPTION
**Pull request by gabyzif**
## Summary

The change ensures that the button styling aligns with the expected design without the need for additional overrides in Nancy.

## Testing Plan

- [x] Was this tested locally? If not, explain why.

| Platform    | Tested |
| -------- | ------- |
| Nancy  |  Yes   |
| Analytics |  Yes    |
| Cortex    |  No   |

Aquarium:
Screenshot 2024-10-15 at 9 56 31 PM

MP:
Screenshot 2024-10-15 at 10 02 44 PM

Styles apllied on MP

Screenshot 2024-10-15 at 10 03 30 PM
Screenshot 2024-10-15 at 10 03 39 PM
Screenshot 2024-10-15 at 10 03 41 PM

Analytics
Screenshot 2024-10-15 at 10 23 41 PM

- Closes https://go.mparticle.com/work/UNI-1113

### CODE REVIEW

**Comment by ilyusha** [comment]
### Review Forge
[Click Here for AI Insights](https://supreme-adventure-m612l2p.pages.github.io?prUrl=https://github.com/mParticle/aquarium/pull/455)

**Comment by gabyzif** [comment]
The reason why it's not being tested in Analytics (yet) is because we can't log in locally to it*

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 1]
```diff
@@ -251,16 +252,12 @@
...
-  font-size: var(--label-size);
-}
-
 .globalNavigation__badge {
   border: solid 1px var(--color-border);
   border-radius: var(--border-radius-sm);
-  padding: 1px var(--padding-content-vertical-sm);
+  padding: 0;
   box-shadow: none !important;
+  margin-left: 4px;
```
Comment (tibuurcio) [inline src/components/navigation/GlobalNavigation/global-navigation.css:260]: Why are we touching badge styles? 🤔

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 2]
```diff
@@ -360,4 +357,47 @@
 .notificationCenter__header {
   padding: var(--padding-content-vertical-lg) var(--padding-content-horizontal-lg);
   border-bottom: 1px solid var(--color-border-secondary);
-}
\ No newline at end of file
+}
+
+/* styles from Nancy */
```
Comment (tibuurcio) [inline src/components/navigation/GlobalNavigation/global-navigation.css:362]: finally! ✨

**Review by tibuurcio — COMMENTED** [review]
Left only 1 question, this also looks good and ready to merge upon testing in Analytics.

**Pull request closed without merging**